main.go

```diff
@@ -86,6 +87,7 @@ func main() {
 	flag.StringSliceVar(&metalbondPeers, "metalbond-peer", nil, "The addresses of the metalbond peers.")
 	flag.BoolVar(&metalbondDebug, "metalbond-debug", false, "Enable metalbond debug.")
 	flag.BoolVar(&tapDeviceMod, "tapdevice-mod", false, "Enable TAP device support")
+	flag.BoolVar(&enableIPv6Support, "enable-ipv6", false, "Enable IPv6 support")
```

Comment:
definitely we need a flag here. but here is a general question: do we support dual stack or ipv4/ipv6-only mode? The answer to this question also has impact on the following control logic changes upon this flag. For the current implementation, two modes exist: v4-only and dual-stack mode. If it is the case, maybe we should be clearer on this point by using something like `enable-ipv6-dualstack`

Reply:
Judging from the relevant PR on the dpservice side, we could support v4-only or v6-only. There is mismatching between supported IP modes here and in dpservice.

Reply:
No. We can support any combination. The flag here enables/disables overall ipv6 support of metalnet.
You can still define the ipv4 address of an interface 0.0.0.0 to disable IPv4 or define the IPv6 Address to "::", so 
you can disable IPv6. Or you can just omit the IPv6 Address in metalnet NetworkInterface object and 
this will set the IPv6 address to "::" behind the scenes when it creates the dpservice side interface.

Reply:
In this case, should both ipv4 and ipv6 rely on the content of the defined addresses, without using an explicit flag? For example, to enable dual stack cluster using kubeadm, no additional flag needs to be provided. Depending on if a ipv4 or ipv6 CIDR range is set or not, a ipv4/ipv6/dualstack cluster is created.

Reply:
This flag should be seen as a feature flag/toggle* to be able to offer a backward compatible metalnet on the top of the fine granular enable/disable on NetworkInterface level. I will add a dpservice version check in the code (Which will then check whether the connected dpservice has a certain version or above when the metalnet feature flag is set) to make the case more understandable.

*https://en.wikipedia.org/wiki/Feature_toggle